### DISCUSSION TITLE
ljj-2022-10-02

### DISCUSSION THREAD

**Discussion by Lcat12345** [MyeongWoonJang/cpp-study#23] [Show and tell]
마크다운 연습 문서 수정
---
![image](https://user-images.githubusercontent.com/62102830/193442272-fde86bf3-80ed-4986-8b5d-20d4d484910c.png)
---
Jira 로드맵에 에픽 추가
---
![image](https://user-images.githubusercontent.com/62102830/193442311-153de2e4-0734-4b48-9f8e-459847da0de5.png)
---
에픽에 하위 이슈들 추가 -> 스프린트에 이슈들 넣기
---
![image](https://user-images.githubusercontent.com/62102830/193442328-2829daea-ab1c-4d88-a929-dc079ae965df.png)
+ 이슈에서 우선순위 삭제후 시작날짜와 기한을 추가해봄
---
워크 플로우 수정
---
![image](https://user-images.githubusercontent.com/62102830/193452494-b514422f-9edf-406e-ac56-3a86a85cc36c.png)
워크 플로우를 건드려 보았습니다.

수정된 워크플로우는 무조건 절차를 따라가도록 설계가 되었습니다.(의도한 바가 아닙니다.)

![image](https://user-images.githubusercontent.com/62102830/193452657-2ef4aa09-547f-480c-9873-d8af8efcf70f.png)

위 이미지를 보시면 해야할 일에서 바로 완료됨을 선택할 수 없게 되었습니다.

기존의 워크 플로우는 모든 상태에서 검토및 피드백, 완료함 등등 상태를 자유롭게 변경할수 있었습니다.

워크 플로 보기에 모든 노드를 "모든 상태의 이슈에서 이 상태로의 이동 허용"을 체크 해보겠습니다.
원래는 모든 노드가 체크되어 있었지만 제가 수정하면서 체크를 전부 해제 했었습니다.
![image](https://user-images.githubusercontent.com/62102830/193452737-b43bc951-21c7-4fde-a4ee-b8f2d780c9da.png)

그러면 다음과 같이 상태를 다시 자유롭게 선택할수 있도록 바꿔집니다.
![image](https://user-images.githubusercontent.com/62102830/193452824-f49035e6-7820-4cd6-badf-d58ecbebf926.png)

그리고 이 워크 플로우는 제 개인적으로 적용되는건 아니고 프로젝트 전체에 적용되는 것 같습니다. 함부로 설정하면 안될것 같은 느낌이네요.

작업 완료 라는 글씨가 보입니까? 이것도 추가한 것입니다.
![image](https://user-images.githubusercontent.com/62102830/193452951-ab950fa7-0544-4f62-8840-fec59cd1ac75.png)
'**전환**'이라고 불리는 것인데 정확히 어떤 역할을 하는 것인지는 모르겟습니다.(역할이라고 부르는게 맞는지도 모르겟습니다.)
![image](https://user-images.githubusercontent.com/62102830/193453043-7d57102f-87af-4eb6-8c07-eb8e1ab04f9e.png)

전환을 모두 삭제시 처음과 같은 상태로 돌아옵니다.
![image](https://user-images.githubusercontent.com/62102830/193453060-a01cdfd5-845a-4d00-8aca-a3fb2cd260f1.png)
![image](https://user-images.githubusercontent.com/62102830/193453067-d5f2a212-3797-4fc0-a5a9-26cc9611699d.png)

**Comment by Woon-2** [comment]
우선순위가 삭제되고 기한이 추가되었군요.
마음껏 건드려주시기 바랍니다.
우선순위는 괜찮은 옵션이라 생각해서 다시 부활했으면 좋겠습니다ㅠ
변경 사항들은 나중에 문서에 반영하겠습니다.

**Reply by Lcat12345** [reply]
워크 플로우를 수정한 내용도 추가했습니다.
그리고 글이 너무 중구난방하다고 느끼는데 피드백 해주시면 감사하겠습니다.

+우선순위 다시 추가하셔도 상관없습니다.

**Reply by Woon-2** [reply]
- #9 에 제가 답변 해놓은 걸 보시면, 이미지를 가운데 정렬하는 방법이 있습니다.
   그것을 참고해, 이미지를 가운데 정렬하면 좋을 것 같습니다.
- 스크럼 내용은 문서가 아니니 체계적일 필요는 없습니다.

  - 저는 항목별로 요약 정리하는 것이 취향이라 이렇게 글을 쓰곤 합니다.

- 지금은 각자가 한 일+할 일을 보고하는 형태이지만, 추후엔 상호 피드백과 협조가 추가된, `스크럼 회의`가 될 것입니다.

  - 짧은 기간이지만, `Discussion`에 이러한 작업 기록을 남기고 보니, 다음과 같은 문제가 있었습니다.
  
    - 다들 본업이 있고, 시간을 내서 하다보니 하루 작업량이 그렇게 많지가 않습니다.
    - 따라서 매일, 혹은 이틀의 주기로 이렇게 글을 쓰게 된다면, 글 쓰는 시간이 실제 작업 시간과 맞먹을 것 같습니다.

  - 따라서, 일주일에 하루 `스크럼 회의`를 진행하는 것이 어떨까 합니다.
   
    - 바로 진행하자는 이야기는 아닙니다.
    - 현재는 좀 느슨하게 작업 기록을 해보고, 한 스프린트가 끝나면 리뷰와 함께 `스크럼 회의`에 대해서 의논해볼까 합니다.

**Reply by Lcat12345** [reply]
뭐든 시도해보는게 좋다고 생각합니다.

피드백 감사합니다.
